lib/wings/active_fedora_converter.rb

```diff
@@ -107,13 +107,23 @@ def initialize(uri = RDF::Node.new, _parent = ActiveTriples::Resource.new)
     private
 
       def convert_members(af_object)
-        return unless resource.respond_to?(:member_ids) && resource.member_ids
+        return unless resource.respond_to?(:member_ids) && !resource.member_ids.blank?
         # TODO: It would be better to find a way to add the members without resuming all the member AF objects
-        ordered_members = []
+        temp_object = assemble_members(af_object)
+        af_object.ordered_member_proxies.association.replace temp_object.ordered_member_proxies.association.to_a
+        af_object.members.proxy_association.target.replace temp_object.members.proxy_association.target
```

Comment:
Not a criticism, but leaving `temp_object` for the GC is something I would imagine maintainers would wish to avoid.  I can't see any alternative to your approach, but it might still be worth noting.